lib/python/ensembl/brc4/runnable/load_sequence_data.py

```diff
@@ -789,13 +874,11 @@ def unversion_scaffolds(self, cs_rank, logs):
                 self.copy_sr_name_to_syn(cs, xdb, self.pjc(logs, "cp2syn", cs))
                 self.sr_name_unversion(cs, "seq_region", "name", self.pjc(logs, "unv_sr", cs))
 
-
     def coord_sys_order(self, cs_order_str):
         cs_order_lst = map(lambda x: x.strip(), cs_order_str.split(","))
-        return { e:i for i,e in enumerate(filter(lambda x: len(x)>0, cs_order_lst)) }
+        return {e: i for i, e in enumerate(filter(lambda x: len(x) > 0, cs_order_lst))}
```

Comment:
prefer to have e:i and i,e together :)
not critical

Reply:
I'm happy to revert particular preferences, but bear in mind `black` will do this change every time it is run over this file :+1:

Reply:
I prefer them separated :P
I'd also prefer to have some space around the { ... } but we can't have everything :D

Reply:
:P

Reply:
if seriously, I don't mind :)

Reply:
@JAlvarezJarreta we need to tune `black` or not use it :)